xep-0313.xml

```diff
@@ -16,6 +16,7 @@
   <dependencies>
     <spec>XMPP Core</spec>
     <spec>XEP-0030</spec>
+    <spec>XEP-0045</spec>
```

Comment:
Actually, I added this dependency because I added a reference to a namespace coming from XEP-0045. However, I agree that it is not a "functional" dependency. I will remove the dependency in the PR, as it seems the way to go.